Paws4Me/HomeCollectionViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  HomeCollectionViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/04/12.
+//
+
+import Foundation
+import UIKit
+
+class HomeCollectionViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate {
+
+    // MARK: - Var/Lets
+    let viewControllerIdentifiers: [String] = ["AllPetDetailViewController", "LocalPetViewController",
```

Comment:
So even though this is relatively logic light this can still live in the `ViewModel` at least that way we can still unit test that the correct count is returned etc...

Reply:
I agree, thank you 👍🏻